server/athenian/api/openapi/openapi.yaml

```diff
@@ -3612,3 +3623,72 @@ components:
       items:
         $ref: '#/components/schemas/FilteredLabel'
       type: array
+    JIRAFilter:
+      additionalProperties: false
+      description: PR filters related to JIRA traits. The aggregation between each
+        filter type is *AND*.
+      example:
+        epics:
+        - DEV-167
+        - DEV-230
+        labels_include:
+        - api,performance
+        - metadata
+        labels_exclude:
+        - feature
+        types:
+        - bug
+        - task
```

Comment:
Just to recap, all the filters are in `AND` while values within each filter are in `OR`, right?

Reply:
Correct!
* It can not be AND inside the epic and issue type filter since a ticket can not be both a bug and a story at the same time. Same for the epic filter where 99% of the time the PR will be mapped to a single epic.
* For the labels, we'll stick to the OR behavior as well at first. After releasing it and receiving feedback from our customers on the GitHub label filter, we'll eventually re-consider this assumption.